psycopg2_extension/management/commands/psqlinit.py

```diff
@@ -0,0 +1,64 @@
+from psycopg2 import sql
+
+from django.core.management.base import CommandError
+from django.db import connections
+
+from psycopg2_extension.base_command import PsycopgBaseCommand
+from psycopg2_extension.utils import init_database, comma_separated_strings
+
+
+class Command(PsycopgBaseCommand):
+    help = 'Initialize the database. It is necessary to use role which has permissions to drop & create databases ' \
+           'and roles.'
+
+    def add_arguments(self, parser):
+        super().add_arguments(parser)
+        parser.add_argument(
+            '--snapshot-file',
+            dest='snapshot_file',
+            help='Snapshot file path'
+        )
+        parser.add_argument(
+            '--extensions',
+            dest='extensions',
+            help='Database extensions separated with ,',
+            type=comma_separated_strings
+        )
+        parser.add_argument(
+            '--noinput', '--no-input', action='store_false', dest='interactive',
+            help='Tells Django to NOT prompt the user for input of any kind.',
+        )
+        parser.add_argument(
+            '--nosnapshot', action='store_true', dest='no_snapshot',
+            help='Do not load snaopshot.',
+        )
+
+    def handle(self, *args, **options):
+        connection = self._get_connection(options['database'])
+
+        if options['interactive']:
+            message = [
+                'This will delete existing database!\n',
+                'Are you sure you want to do this?\n\n'
+                "Type 'yes' to continue, or 'no' to cancel: "
+            ]
```

Comment:
why is there a comma at the first line but not the others?

Reply:
You are true, no comma isrequired here